rustls/src/webpki/client_verifier.rs

```diff
@@ -247,13 +240,32 @@ pub struct WebPkiClientVerifier {
 
 impl WebPkiClientVerifier {
     /// Create builder to build up the `webpki` client certificate verifier configuration.
+    ///
     /// Client certificate authentication will be offered by the server, and client certificates
     /// will be verified using the trust anchors found in the provided `roots`. If you
     /// wish to disable client authentication use [WebPkiClientVerifier::no_client_auth()] instead.
     ///
+    /// The cryptography used comes from the default [`CryptoProvider`]: [`crate::crypto::ring::RING`].
+    /// Use [`Self::builder_with_provider`] if you wish to customise this.
+    ///
     /// For more information, see the [`ClientCertVerifierBuilder`] documentation.
+    #[cfg(feature = "ring")]
     pub fn builder(roots: Arc<RootCertStore>) -> ClientCertVerifierBuilder {
-        ClientCertVerifierBuilder::new(roots)
+        Self::builder_with_provider(roots, crate::crypto::ring::RING)
+    }
+
+    /// Create builder to build up the `webpki` client certificate verifier configuration.
```

Comment:
If the other headline were changed this one could stay as it is and would probably be clear enough with the `_with_provider` name. Right now both fns have the same headline and I think that could be made nicer.